docs/_posts/murat-gunay/2022-04-08-bert_biolink_large_en_3_0.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: model
+title: BERT Biolink Embeddings
+author: John Snow Labs
+name: bert_biolink_large
+date: 2022-04-08
+tags: [biolink, bert, medical, en, open_source]
+task: Embeddings
+language: en
+edition: Spark NLP 3.4.2
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This embeddings is trained on PubMed abstracts along with citation link information. It is introduced in the this [paper](https://arxiv.org/abs/2203.15827). This model achieves state-of-the-art performance on several biomedical NLP benchmarks such as [BLURB](https://microsoft.github.io/BLURB/) and [MedQA-USMLE](https://github.com/jind11/MedQA).
```

Comment:
This embeddings component was trained on PubMed abstracts all along with citations link information. The embeddings were introduced in this paper ....
(all the rest is ok)